h264/pps.go

```diff
@@ -49,27 +49,27 @@ func NewPPS(sps *SPS, rbsp []byte, showPacket bool) *PPS {
 		return false
 	}
 
-	pps.ID = ue(b.golomb())
-	pps.SPSID = ue(b.golomb())
+	pps.ID, _ = readUe(nil)
```

Comment:
Why are the errors dropped?

Reply:
This is intended to be temporary. Should I do add a TODO, or actually handle the errors now ?

Reply:
The errors should be handled now.